packages/plugin-loyalties-ui/src/configs/scoreCampaign/components/List.tsx

```
@@ -0,0 +1,193 @@
+          <th>{__("Action")}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {campaigns.map((campaign) => (
+          <Row
+            toggleBulk={toggleBulk}
+            isChecked={bulk.includes(campaign._id)}
+            campaign={campaign}
+            refetch={refetch}
+            onChangeStatus={onChangeStatus}
+          />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)